Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [workspace]
+resolver = "2"
```

Comment:
I think it doesn't need because this has been the default resolver since Rust 2021 edition.

Reply:
It produces warning. This is the workspace `cargo.toml`, so it doesn't have the `Rust 2021 edition` definition. Therefore the `resolver = "2"2` is required here to be explicit. I don't think this changes anything for us other than making the warning goes away, because all of our crates are 2021 edition. However this warning is geared to people having a mixed editions for different crates in the workspace.

Specifically here are the a few reference for this issue:

This reference points to we need to explicitly set resolver in the workspace cargo.toml: https://github.com/rust-lang/edition-guide/pull/267/files#diff-e24ffee7de69b814b2c59439c6cfc9b8f9978b864175fd4425911b897391d845R17

One issue I found for the specific problem: https://github.com/rust-lang/cargo/issues/9956

The specific lint rule is added with this issue: https://github.com/rust-lang/cargo/issues/10112